src/components/Themeroller/ThemeTuner/__tests__/ThemeTuner.spec.tsx

```diff
@@ -88,21 +89,16 @@ describe('ThemeTuner component', () => {
         expect(view.container).toMatchSnapshot();
     });
 
-    // TODO can't change value
-    it.skip('should invoke onChange callback when first input was changed', async () => {
+    it('should invoke onChange callback when first input was changed', async () => {
+        const onChangeMock = jest.fn();
         const user = userEvent.setup();
-        const view = render(
+        view = render(
             <ThemeTuner config={ThemeRollerTestConfig} cssVars={{}} onChange={onChangeMock} />
         );
-
         expect(view.container).toMatchSnapshot();
 
-        const firstInput = screen.getByRole('spinbutton');
-
-        await user.type(firstInput, '#00000');
+        await user.type(screen.getByRole('textbox'), '#ffffff');
 
-        expect(onChangeMock).toHaveBeenCalledWith({
-            '--color-background': '#00000',
-        });
+        expect(onChangeMock).toHaveBeenCalled();
```

Comment:
Can we use `toHaveBeenCalledWith`?

Reply:
It shows 
![image](https://github.com/textkernel/oneui/assets/104063210/a74880b0-103f-439b-b6b0-a1840a0a8023)

Reply:
hmm, then something is wrong with the test itself.

Reply:
I think the test is correct. The main problem is that enzyme allow to set the hole value to input. But when we use RTL type function it types by element here. The same problem we have for SelectBase.spec.tsx it was 
![image](https://github.com/textkernel/oneui/assets/104063210/2da0a2e5-25dd-45d9-9d24-a5ed754269c5)

And then we replace it to 
![image](https://github.com/textkernel/oneui/assets/104063210/122849dd-b670-43e5-8dce-dca08b0c1292)

Reply:
The solution is to use fireEvent.change I'll apply it for both tests